docs/connecting-data/connecting-to-apis-and-external-data/jira.md

```diff
@@ -44,62 +169,79 @@ You can access your Organization at [https://admin.atlassian.com/](https://admin
 
 
 
-### Query Page Sidebar
+### Nav Tree
+The endpoints above will display as follows in the nav tree once your API has successfully connected.
+
+<figure markdown>
+  ![JIRA Endpoints][image-2]{ width="100%" }
+  <figcaption>Query Page Sidebar</figcaption>
+</figure>
 
-![JIRA Endpoints][image-2]
 
 ## Sample Queries
+Suppose that my JIRA API data source was called `myjiraapi` and I want to query an endpoint. The endpoint goes after the JIRA data source name like so: `#!sql FROM myjiraapi.<ENDPOINT>`.
 
```

Comment:
Move this under the text on line 184.

Change wording: For the following examples, suppose that my JIRA API data source was called... 

Insert back ticks into the example around the data source and endpoint